jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdConfiguration.java

```diff
@@ -97,14 +99,38 @@ public OpenIdConfiguration(@Name("issuer") String issuer,
                                @Name("clientSecret") String clientSecret,
                                @Name("authMethod") String authMethod,
                                @Name("httpClient") HttpClient httpClient)
+    {
+        this(issuer, authorizationEndpoint, tokenEndpoint, null, clientId, clientSecret, authorizationEndpoint, httpClient);
```

Comment:
I think `authorizationEndpoint` is passed by accident instead of `authMethod` (`authorizationEndpoint` is passed twice)

Reply:
Good catch, thanks! Fixed.